docs/notebooks/extendxclim.ipynb

```diff
@@ -8,29 +8,43 @@
     "\n",
```

Comment:
<p>extra "(e" at the end. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>